lib/rig_inbound_gateway_web/v1/connection_controller.ex

```diff
@@ -0,0 +1,76 @@
+defmodule RigInboundGatewayWeb.V1.ConnectionController do
+  @moduledoc false
+  use Rig.Config, [:cors]
+  use RigInboundGatewayWeb, :controller
+  use RigInboundGatewayWeb.Cors, :cors
+
+  # TODO: Find a nicer way to do this...maybe `use RigInboundGatewayWeb.Cors, :put, :delete` ?
+  use RigInboundGatewayWeb.Cors, :preflight_all
+
+  alias Result
+  alias Rig.Connection.Codec
+  alias RigInboundGatewayWeb.VConnection
+
+  require Logger
+
+  @heartbeat_interval_ms 15_000
+  @subscription_refresh_interval_ms 60_000
+
+  @doc """
+  ### Dirty Testing
+
+      CONN_TOKEN=$(http :4000/_rig/v1/connection/init)
+      http --stream ":4000/_rig/v1/connection/sse?connection_token=$CONN_TOKEN"
+  """
+  @spec init(conn :: Plug.Conn.t(), params :: map) :: Plug.Conn.t()
+  def init(%{method: "GET"} = conn, _) do
+    {:ok, vconnection_pid} = VConnection.start(@heartbeat_interval_ms, @subscription_refresh_interval_ms)
+    send vconnection_pid, :vconnection_timeout
```

Comment:
why does the VConnection require its timer to be triggered from outside and doesn't start it automatically?